frontend/src/router/index.js

```diff
@@ -7,14 +7,25 @@ import NotFound from "../pages/NotFound";
 
 const DEFAULT_PAGE = "/giphy";
 
-const Router = () => {
-  return (<Routes>
-    {APIS.map(({ apiType }) => 
-      <Route key={apiType} path={`/${apiType}`} element={<GalleryPage />} />
-    )}
-    <Route path="/" element={<Navigate to={DEFAULT_PAGE} replace />} />
-    <Route path="*" element={<NotFound />} />
-  </Routes>);
+const Router = ({ searchWord, setSearchWord }) => {
+  return (
+    <Routes>
+      {APIS.map(({ apiType }) => (
+        <Route
+          key={apiType}
+          path={`/${apiType}`}
+          element={
+            <GalleryPage
+              inputValue={searchWord}
```

Comment:
please keep the states name same through all components.  this should have same name as on other components `searchValue`